gossip/gossip.go

```diff
@@ -777,14 +781,14 @@ func (g *Gossip) bootstrap() {
 			select {
 			case <-bootstrapTimer.C:
 				bootstrapTimer.Read = true
-				// continue
+				// break
```

Comment:
I don't think the intention was to continue in this case as then we would never reach the second select statement. Perhaps the original author was trying to document the control flow and was indicating "continue with the rest of the loop".